src/main/resources/com/atlassian/jira/cloud/jenkins/configuration/config.jelly

```diff
@@ -1,8 +1,158 @@
+                                                <td>
+                                                    <span id="webhookUrl_${loop.index}">${site.webhookUrl}</span>
+                                                </td>
+                                                <td class="text-align-right">
+                                                    <a href="#" class="margin-right-1em" onclick="editSite(${loop.index}); return false;">
+                                                        <l:icon src="symbol-edit" class="icon-sm" tooltip="Edit Site details"/>
+                                                    </a>
+                                                    <a href="#" onclick="removeSite(${loop.index}); return false;">
+                                                        <l:icon src="symbol-trash" class="icon-sm" tooltip="Remove Site"/>
+                                                    </a>
+                                                </td>
+                                            </tr>
+                                        </j:forEach>
+                                    </tbody>
+                                </table>
+                            </f:section>
+                        </div>
+
+                        <!-- Add Site Button -->
+                        <input type="button" id="showSiteButton" value="Add Site" class="margin-bottom-2em" onclick="showSiteInputs();" />
+
+                        <div id="siteDataContainer" class="site-data-default" name="sites">
+                            <f:section title="Site Connection details">
+                                <div class="help-area">
+                                    <div class="help display-block">
+                                        <div>A Jira admin will provide the credentials needed to connect to this site.</div>
+                                        <div>Once you've entered these details, select <b>Save</b> to connect.</div>
+                                    </div>
+                                </div>
+
+                                <f:entry title="Site Name:" help="/plugin/atlassian-jira-software-cloud/help-siteName.html" field="site">
+                                    <f:textbox name="site" default="sitename.atlassian.net" />
+                                </f:entry>
+
+                                <f:entry title="Webhook URL:" help="/plugin/atlassian-jira-software-cloud/help-webhookUrl.html" field="webhookUrl">
+                                    <f:textbox name="webhookUrl"  />
+                                </f:entry>
+
+                                <f:entry class="com.atlassian.jira.cloud.jenkins.configuration" title="Secret:" help="/plugin/atlassian-jira-software-cloud/help-webhookSecret.html">
+                                    <c:select field="credentialsId" />
+                                </f:entry>
+
+                                <input type="hidden" name="active" value="false" />
```

Comment:
the one manipulation we do, do, is flag the edit/new site form with a active flag, on the server side we exclude active=false entries, this is commented in the management link code and does the deed wonderfully